tools/docker/Dockerfile

```diff
@@ -116,6 +117,16 @@ RUN wget -nv https://github.com/renode/renode/releases/download/v${RENODE_VERSIO
   && rm renode_${RENODE_VERSION}_amd64.deb \
   && apt-get -qq clean
 
+ARG COMMANDER_VERSION=1v13p1b1154
+RUN wget -nv https://www.silabs.com/documents/public/software/SimplicityCommander-Linux.zip && \
```

Comment:
This file now contains a tar.bz2 of version 1v13p3b1203, so the version can't be hardcoded in the Dockerfile.

Reply:
Yes, that is not a good way to version control. It prevents us from fixing a specific commander version. I have pushed a fix for it.